_inc/client/my-plan/my-plan-header/checklist-cta.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { translate as __ } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+
+export default function ChecklistCta( { siteSlug } ) {
+	return (
+		<div className="jp-landing__plan-features-header-checklist-cta-container">
+			<Button primary href={ `https://wordpress.com/plans/my-plan/${ siteSlug }` }>
+				{ __( 'View your setup checklist' ) }
```

Comment:
Is the `__` alias going to work fine here? :thinking: It will probably be handled by the plugin repo's string extraction handling, which is made for `@wordpress/i18n`, and the string will end up in the translation files generated by the plugin repo. But `i18n-calypso` won't load those, will it?

Reply:
This is nothing new or novel, it's how things are done around here 🙂 

https://github.com/Automattic/jetpack/blob/510a46bda82d61245e10f392933a09d839218871/_inc/client/my-plan/my-plan-body.jsx#L8
https://github.com/Automattic/jetpack/blob/510a46bda82d61245e10f392933a09d839218871/_inc/client/my-plan/my-plan-body.jsx#L129

Reply:
I think there's a script somewhere which pulls these out to a php file, and we're not relying on wp.org process to extract these. 🤔 @zinigor might know better.

Reply:
The magic happens here:
https://github.com/Automattic/jetpack/blob/7b6c7495020716d3d89fd7b0b8d0f4f3093348bb/gulpfile.babel.js#L204

Reply:
TIL